.github/workflows/deploy-website.yml

```diff
@@ -8,8 +8,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
https://github.com/actions/checkout
>    \# Number of commits to fetch. 0 indicates all history for all branches and tags.
>    \# Default: 1

Reply:
I don't understand why this number is wrong ...but let's try!

Reply:
For the record the release job gets proper versions because it does an explicit unshallow https://github.com/scalacenter/scalafix/blob/6d991eb0bcaf4afd635c3c1657af59e946548ea0/.github/workflows/release.yml#L13

I guess either approach is correct. The unshallow is maybe a bit more efficient but I like that `fetch-depth` is a flag on the checkout as it's more apparent.